sdk/test-utils/recorder-new/src/utils/env.browser.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// In the browser, we load the env variables with the help of karma.conf.js
+
+export const env = (window as any).__env__;
```

Comment:
Should this throw with a descriptive message if karma config did not load the env vars?

Reply:
This file shouldn't know if the karma.conf has loaded the variables or not.

Reply:
If a certain variable is undefined, we'll know from the test failure anyways.